### PR TITLE
Redpills chemical colors

### PR DESCRIPTION
**Pull request by pudl**
:cl:
rscadd: Chemicals should now have color, instead of just being pink.
/:cl:

### CODE REVIEW

**Comment by pudl** [comment]
This is a usability tweak not a feature

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -680,7 +680,7 @@
 	name = "Sulphuric acid"
 	id = "sacid"
 	description = "A strong mineral acid with the molecular formula H2SO4."
-	color = "#DB5008" // rgb: 219, 80, 8
+	color = rgb(0,255,50)
```
Comment (ChangelingRain) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: use hex color codes!!!

Comment (pudl) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: No

**Comment by pudl** [comment]
What the fuck

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: okay let me rephrase USE HEX COLOR CODES

Comment (pudl) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: No

**Comment by Jordie0608** [comment]
Reagents already have colors
capture

**Comment by pudl** [comment]
Jordie this gives color to the ones that goof left pink

**Comment by Jordie0608** [comment]
They're not all pink; there's a range of blue, greens and reds among others in what you're changing

**Comment by pudl** [comment]
These colors are far more redpilled

**Comment by Iamgoofball** [comment]
@Jordie0608 i would of made them these colors if I had the goonmed colors originally

**Comment by Cheridan** [comment]
use hex color codes

**Comment by Iamgoofball** [comment]
no

**Comment by Thunder12345** [comment]
Use hex colour codes

**Comment by xxalpha** [comment]
:one: :six:

**Comment by pudl** [comment]
@Cheridan Remove the dnm tag

**Comment by ChangelingRain** [comment]
Used hex color codes!

**Comment by Sarcalogo** [comment]
dude what the fuck do you mean when you say redpill. I am legitimately confused. are you like taking this red pill /pol/ meme far too seriously?? I don't fucking get it.

**Comment by pudl** [comment]
@Sarcalogo Dont comment to me or my son ever again
